pagopa/io-app
# [SIW-1029] Add IT Wallet credential preview screen

> [!WARNING]
> This PR depends on #5832 

## Short description
This PR implements the IT Wallet credential preview screen which allows the user to see the data of a credential before it will be stored in the device.

> [!NOTE]
> This screen is currently using mocked **mDL** data and does not represent the final implementation of the component, which will come in a later PR.

## List of changes proposed in this pull request
- Added `ItwIssuanceCredentialPreviewScreen` component
- Added required routes, parameters and navigation components
- Added `ItwCredentialPreviewScreenContent` component which renders the preview of a credential. This component will be used inside bot the eID and credential preview screen, which are two separate flows but share the same UI.

## How to test
Navigate to **Profile > Playgrounds > IT Wallet > Credential preview**. Check that everything is being rendered correctly.

## Previews

| iOS | Android |
| --- | --- |
|  <video src="https://github.com/pagopa/io-app/assets/6160324/aae69f8a-eada-48dd-9332-e47e416ca259" /> |  <video src="https://github.com/pagopa/io-app/assets/6160324/437a0b4d-863b-4e79-8772-0831d213a5ad" />
